src/CoreWCF.WebHttp/src/CoreWCF/Channels/WebBodyFormatMessageProperty.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Globalization;
+
+namespace CoreWCF.Channels
+{
+    public sealed class WebBodyFormatMessageProperty : IMessageProperty
+    {
+        public const string Name = "WebBodyFormatMessageProperty";
+
+        private static WebBodyFormatMessageProperty _jsonProperty;
+        private static WebBodyFormatMessageProperty _xmlProperty;
```

Comment:
Naming convention should be s_ as static

Reply:
Addressed in 8722a911f6ec91bff5803c85cfe9082240d1c360.